services/order.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth import get_user_model
+from db.models import Order, Ticket
+from django.db import transaction
+from django.db.models import QuerySet
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                order_id=order.id,
+                row=ticket["row"],
+                seat=ticket["seat"]
+            )
+        order.save()

```

Comment:
You made save() in if check. If you don't change anything after creation no need to use save() more.